docs/Packaging.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Packaging
+layout: docs
+---
+
+Theos supports building for many common targets and file formats including:
+
+### Platforms
+- iOS
+- macOS
+- Linux
+- Windows
+
+### File formats
+- .dylib
+- .a
+- .so
+- .deb
+- .ipa
+- null packages
+- .pkg
+- .rpm
+- .exe
+- .dll
```

Comment:
How come executable formats are mixed with packaging formats here?

Reply:
I can't remember if there was a specific reason ... likely just got carried away and listed all of the file formats I came across. Whoops!

Reply:
Removed non-packaging formats from the list in a recent commit to #19.